### PR TITLE
{bio}[fosscuda/2019b] GROMACS v2019.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3301.joltik.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz, Python 2.7.5
See https://gist.github.com/14e08c54e3d9e55d5e1d44aa9b334a1e for a full test report.

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2019.4-fosscuda-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+name = 'GROMACS'
+version = '2019.4'
+
+homepage = 'http://www.gromacs.org'
+description = """
+GROMACS is a versatile package to perform molecular dynamics,
+ i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.
+
+This is a CPU only build, containing both MPI and threadMPI builds.
```
Comment (akesandgren) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2019.4-fosscuda-2019b.eb]: This line is wrong. Please see the other fosscuda versions.

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2019.4-fosscuda-2019b.eb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = [
+    'https://ftp.gromacs.org/pub/gromacs/',
+    'ftp://ftp.gromacs.org/pub/gromacs/',
+]
+sources = [SOURCELOWER_TAR_GZ]
+patches = [
+    'GROMACS-2019_fix_omp_num_threads_and_google_test_death_style_in_tests.patch',
+    'GROMACS-2018_dont_do_gpu_tests_on_cpu_build.patch',
```
Comment (akesandgren) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2019.4-fosscuda-2019b.eb]: Missing, GROMACS-2019_increase_test_timeout_for_GPU.patch
don't need GROMACS-2018_dont_do_gpu_tests_on_cpu_build.patch

**Comment by boegel** [comment]
Not going to pursue this, superseded by more recent GROMACS versions with CUDA support, like #14136

**Pull request closed without merging**